src/pages/home.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { useSelector } from 'react-redux';
+import Header from '../components/Header';
+import Characters from '../components/Characters';
+
+const Home = () => {
+  const { characters } = useSelector((store) => store.character);
+  const [filter, setFilter] = useState('');
+
+  const handleChange = (e) => {
+    setFilter(e.target.value);
+  };
+
+  const filteredCharacters = characters.filter(
+    (character) => character.name.toLowerCase().includes(filter.toLowerCase()),
+  );
+
+  return (
+    <Wrapper>
+      <section>
+        <Header />
+        <div className="input">
+          <form className="form">
+            <input type="text" placeholder="Enter name of the character" onChange={handleChange} />
+          </form>
+        </div>
+        <Characters characters={filteredCharacters} />
+      </section>
+    </Wrapper>
+  );
+};
```

Comment:
- [ ] As required by the requirements follow the same spacing, layouts, and design so you should be to remove. the spaces between cards 
- screenshot of the requirements
<img width="544" alt="Screenshot 2023-04-19 at 22 03 36" src="https://user-images.githubusercontent.com/61790587/233209985-3212ef48-1f35-4881-8793-12b5ed137017.png">

- Yours 
<img width="295" alt="Screenshot 2023-04-19 at 21 58 51" src="https://user-images.githubusercontent.com/61790587/233209901-96b60fc0-6baa-444a-9ec4-61030c731ac7.png">
- Design 
<img width="206" alt="Screenshot 2023-04-19 at 22 03 15" src="https://user-images.githubusercontent.com/61790587/233209935-209a95ac-eede-4ff8-947d-1900c708c11c.png">